src/components/ActionBar.vue

```diff
@@ -1035,7 +1012,7 @@ div.bar-filter-wrapper
   &:last-child
     border-radius: 0 0 6px 6px !important
 .q-scrollarea
-  height: 200px
+  height: 60vh
```

Comment:
🫶

Reply:
Again, this change was not requested so I can revert it if you prefer.
What was made here is to make more visible the dialog when using mobile.

![image](https://user-images.githubusercontent.com/4420760/194327009-365b7d8a-80d3-411a-a7bc-b9fa2b11d3bd.png)

Reply:
Again, appreciation because makes the experience responsive so muuuuuch better
<img width="286" alt="Screen Shot 2022-10-06 at 20 45 00" src="https://user-images.githubusercontent.com/2939354/194463571-91619d6f-ed78-4b6b-ac91-597afe2b58e2.png">